cdap-ui/app/cdap/components/WizardModal/WizardModal.scss

```diff
@@ -16,15 +16,19 @@
 
 @import '../../styles/variables.scss';
 $wizard-header-color: #464a57;
-$wizard-modal-margin-top: 180px;
+$wizard-modal-height: 85vh;
+$wizard-modal-width: 60vw;
+$wizard-modal-margin-top: 12.5vh;
+$wizard-modal-body-max-height: 360px;
+$wizard-modal-max-width: 1050px;
 
 .wizard-modal {
   overflow-x: hidden;
   &.modal-dialog {
-    height: 65vh;
-    width: 60vw;
+    height: $wizard-modal-height;
```

Comment:
@elfenheart I played with changing around these values for a bit. The `max-height` on the `.modal-body` is necessary, because otherwise it would just expand and take up the space needed for the resizable error message.